api/lib/application/badges/index.js

```diff
@@ -49,10 +49,10 @@ exports.register = async function (server) {
                 key: Joi.string().required(),
                 'alt-message': Joi.string().required(),
                 'image-url': Joi.string().required(),
-                message: Joi.string().required().allow('').allow(null),
-                title: Joi.string().required().allow('').allow(null),
-                'is-certifiable': Joi.boolean().required(),
-                'is-always-visible': Joi.boolean().required(),
+                message: Joi.string().required().allow(null),
+                title: Joi.string().required(),
```

Comment:
**question** : je me demande si en vrai il y a pas des badges sans titre, à checker

Reply:
J'ai enlevé les `.allow('')` et `.allow(null)` cas dans le hbs , ce champ est obligatoire et `required={{true}}` , donc ne peut pas être vide ou null. voila le champs dans le fichier badge.hbs
`<div class="badge-edit-form__field">
              <label for="title" class="badge-edit-form-field__label">
                <abbr title="obligatoire" class="mandatory-mark">*</abbr>
                Titre :
              </label>
              <Input id="title" @type="text" class="form-control" @value={{this.form.title}} required={{true}} />
            </div>`